lib/RequestHandler.ts

```diff
@@ -0,0 +1,52 @@
+/// <reference path="../typings/tsd.d.ts" />
+import Promise = require('bluebird');
+import restify = require('restify');
+import apiSession = require('./api-session');
+import BAPI = require('./blpapi-wrapper');
+
+export = RequestHandler;
+
+class RequestHandler {
+    private blpSession: BAPI.Session;
+
+    constructor(blpsess: BAPI.Session) {
+        this.blpSession = blpsess;
```

Comment:
What happens when `blpsess` emits a `SessionTerminated` event?  It seems there is an attempt [here](https://github.com/ericlu88/blpapi-http/commit/4c449c45dce11e653116b1fef5634b048d12b224#diff-ed009b6b86b017532ef0489c77de5100R79).  However, that reassigns the global variable in `index.ts` without resetting the `blpSession` member variable within an instance of a `RequestHandler`.